### PR TITLE
Dev instance docs

### PR DESCRIPTION
**Pull request by victoria-miltcheva**
**What does this PR do?**
- `kustomize_envs/dev/kustomization.yaml`: points to correct `gd_db.conf` location to fix local kube env deploy
- `kustomize_envs/dev/README.md`: adds in comment about how to set up manual local dev secrets and removes ibm-specific references

### CODE REVIEW

**Review by killuazhu — DISMISSED** [review]

**Comment by killuazhu** [comment]
oops... too eager to approve.  Did not notice it's in draft. 😄

**Comment by victoria-miltcheva** [comment]
@killuazhu No problem! The reason I have it set as a draft is because when I run `skaffold dev --port-forward -p dev`, it fails to create the `gd-ingest` and `scan-worker` deploys:
```
 - dev:deployment/gd-revoker is ready. [2/5 deployment(s) still pending]
 - dev:deployment/gd-ingest: creating container gd-ingest
    - dev:pod/gd-ingest-5bf8bcbd9-gb5b6: creating container gd-ingest
 - dev:deployment/gd-ingest failed. Error: creating container gd-ingest.
 - dev:deployment/scan-worker: creating container scan-worker
    - dev:pod/scan-worker-f6bc566fc-7hc9z: creating container scan-worker
 - dev:deployment/scan-worker failed. Error: creating container scan-worker
```

I've run into this error before, but can't remember how i resolved it so I've been troubleshooting it. It might not even be related to the repo itself. When you have a moment would you mind testing to see if you can successfully run the local kube cluster with the changes from this branch?

File: kustomize_envs/dev/README.md [hunk 1]
```diff
@@ -4,14 +4,14 @@ This doc describes how to use `skaffold` to run a detect-secrets-stream instance
...
 ## Dependencies
 
-### provision ahead of time (already done)
+### provision ahead of time
 
-- github-app: [detect-secrets-admin-tool-test](https://github.ibm.com/github-apps/detect-secrets-admin-tool-test)
-  - `kustomize_envs/dev/secret/app.key` stores the private key for this test Github App
-  - `APP_ID` in `kustomize_envs/dev/secret/env.txt` stores the Github App ID
+- github-app:
+  - Named `detect-secrets-admin-tool-test`
```
Comment (killuazhu) [inline kustomize_envs/dev/README.md:10]: This can be any name.

**Review by killuazhu — COMMENTED** [review]
One comment on github app name.

**Comment by victoria-miltcheva** [comment]
Update: I'm now able to spin up the local kube cluster config. The issue was that the `email_conf` file needed to be listed: https://github.com/IBM/detect-secrets-stream/pull/7/commits/2e6de683d567814e37b3f24585093e49cc7038e0.

File: kustomize_envs/dev/README.md (hunk 1 above)
Comment (victoria-miltcheva) [reply kustomize_envs/dev/README.md]: https://github.com/IBM/detect-secrets-stream/pull/7/commits/d6b77a82f39a2cd4b547f713d9aa78cff646adb7

**Review by killuazhu — APPROVED** [review]
> The issue was that the email_conf file needed to be listed

Hahaha, you should know that very well since you wrote it. 😛 

Thanks for making the PR!

**Comment by victoria-miltcheva** [comment]
> > The issue was that the email_conf file needed to be listed
> 
> Hahaha, you should know that very well since you wrote it. 😛
> 
> Thanks for making the PR!

Better to have caught it now than later 😆 . Thanks for your review!

**Pull request merged**